spec/view_helper/links_spec.rb

```diff
@@ -122,4 +122,25 @@
       end
     end
   end
+
+  context 'displaying image_src url' do
+    it 'should not display image_src url by default' do
+      subject.display_meta_tags(site: 'someSite').tap do |meta|
+        expect(meta).to_not have_tag('link', with: { href: "http://example.com/base/url", rel: "image_src" })
```

Comment:
Line is too long. [109/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.